build.gradle.kts

```diff
@@ -9,6 +9,7 @@ version = "1.0-SNAPSHOT"
 
 repositories {
     mavenCentral()
+    maven("https://dl.bintray.com/cbeust/maven")
```

Comment:
Might be better to move that below JitPack. I am not completely sure if there are any restrictions on private Bintray Maven repositories, but if that is not the case the owner of that repository could (for whatever reason) publish modified 3rd party artifacts which we are currently retrieving from JitPack.